packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <p
+        :class="{
+            [$style['c-restaurantCard-iconText']]: true,
+            [$style['c-restaurantCard-iconText--bold']]: isBold
+        }">
+        <span
+            v-if="showIcon"
+            :class="{
+                [$style['c-restaurantCard-iconText-icon']]: true,
+                [$style[`c-restaurantCard-iconText-icon--${color}`]]: true,
+                [$style['c-restaurantCard-iconText-icon--hideOnMidBelow']]: hideIconInTileView
+            }"
+            data-test-id="icon-text-icon">
+            <slot />
+        </span>
+        <span
+            :class="{
+                [$style['c-restaurantCard-iconText-content']]: true,
+                [$style[`c-restaurantCard-iconText-content--${color}`]]: true
+            }"
+            data-test-id="icon-text-visible-text">
+            {{ text }}
+        </span>
+    </p>
+</template>
+
+<script>
+export default {
+    name: 'IconText',
+    inject: {
+        isListItem: {
+            default: false
+        }
+    },
+    props: {
+        text: {
+            type: String,
+            required: true
+        },
+        color: {
```

Comment:
One suggestion for a future improvement to this – I'd probably have some validation over what colours this can be. That way there's a known set of colours that the icon can be set as (because otherwise, passing in blue atm, it won't actually make it blue, it'll just add a class that doesn't do anything I think?)

Sorry for the delayed response – just been trying to get to inbox zero due to the OneApp stuff that was going on recently!